src/sentry/integrations/github/webhook.py

```diff
@@ -90,7 +91,23 @@ def __call__(self, event: Mapping[str, Any], host: str | None = None) -> None:
                 provider=f"integrations:{self.provider}",
                 external_id=str(event["repository"]["id"]),
             )
-            for repo in repos:
+
+            if not repos.exists():
+                provider = get_integration_repository_provider(integration)
+
+                config = {
+                    "integration_id": integration.id,
+                    "external_id": str(event["repository"]["id"]),
+                    "identifier": event.get("repository", {}).get("full_name", None),
+                }
+
+                for org in list(orgs.values()):
```

Comment:
nit: you can iterate directly
```suggestion
                for org in orgs.values():
```